tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -45,6 +45,7 @@ func TestStatusExistsByDefault(t *testing.T) {
 }
 
 func TestAnalysisWritesStatus(t *testing.T) {
+	t.Skip("Re-enable once https://github.com/istio/istio/issues/26229 is resolved")
```

Comment:
Are we sure the problem is the test and not the feature?  I don't see any explicit calls to client-go here...

Reply:
not skipping in the fix.